libraries/radpdfprocessing/concepts/imagequality.md

```diff
@@ -63,6 +63,10 @@ When **ImageQuality** of an image is set to **High**, **RadPdfProcessing** inter
 
 > JPEG2000 images in **RadPdfProcessing** can be inserted only with **ImageQuality.High**. Exporting them with lower ImageQuality value requires decoding JPEG2000 files, which is currently unsupported by the library. 
 
+### ImageQuality .NET Standard
+
+.NET Standard specification does not define APIs for converting images or scaling their quality. Thats why, to allow the library to export images different than Jpeg and Jpeg2000 or ImageQuality different than High, you will need to provide an implementation of the **JpegImageConverterBase** abstract class and set this implementation to the **JpegImageConverter** property of the of **FixedExtensibilityManager**. For more information check the [Cross-Platform Support]({%slug radpdfprocessing-cross-platform%}) help article.
```

Comment:
### ImageQuality in .NET Standard

That is why to allow the library to export images different than Jpeg and Jpeg2000 or ImageQuality different than High, you will need to provide an implementation of the **JpegImageConverterBase** abstract class. This implementation should be passed to the **JpegImageConverter** property of **FixedExtensibilityManager**.